Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/MarkdownTextBlock.cs

```diff
@@ -0,0 +1,1076 @@
+            set { SetValue(CodeBorderBrushProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets the dependency property for <see cref="CodeBorderBrush"/>.
+        /// </summary>
+        public static readonly DependencyProperty CodeBorderBrushProperty = DependencyProperty.Register(
+            nameof(CodeBorderBrush),
+            typeof(Brush),
+            typeof(MarkdownTextBlock),
+            new PropertyMetadata(new SolidColorBrush(), new PropertyChangedCallback(OnPropertyChangedStatic)));
+
+        /// <summary>
+        /// Gets or sets the thickness of the border around code blocks.
+        /// </summary>
+        public Thickness CodeBorderThickness
+        {
+            get { return (Thickness)GetValue(CodeBorderThicknessProperty); }
+            set { SetValue(CodeBorderThicknessProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets the dependency property for <see cref="CodeBorderThickness"/>.
+        /// </summary>
+        public static readonly DependencyProperty CodeBorderThicknessProperty = DependencyProperty.Register(
+            nameof(CodeBorderThickness),
+            typeof(Thickness),
+            typeof(MarkdownTextBlock),
+            new PropertyMetadata(new Thickness(2), new PropertyChangedCallback(OnPropertyChangedStatic)));
+
+        /// <summary>
+        /// Gets or sets the brush used to render the text inside a code block.  If this is
+        /// <c>null</c>, then Foreground is used.
+        /// </summary>
+        public Brush CodeForeground
+        {
+            get { return (Brush)GetValue(CodeForegroundProperty); }
+            set { SetValue(CodeForegroundProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets the dependency property for <see cref="CodeForeground"/>.
+        /// </summary>
+        public static readonly DependencyProperty CodeForegroundProperty = DependencyProperty.Register(
+            nameof(CodeForeground),
+            typeof(Brush),
+            typeof(MarkdownTextBlock),
+            new PropertyMetadata(null, new PropertyChangedCallback(OnPropertyChangedStatic)));
+
+        /// <summary>
+        /// Gets or sets the font used to display code.  If this is <c>null</c>, then
+        /// <see cref="FontFamily"/> is used.
+        /// </summary>
+        public FontFamily CodeFontFamily
+        {
+            get { return (FontFamily)GetValue(CodeFontFamilyProperty); }
+            set { SetValue(CodeFontFamilyProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets the dependency property for <see cref="CodeFontFamily"/>.
+        /// </summary>
+        public static readonly DependencyProperty CodeFontFamilyProperty = DependencyProperty.Register(
+            nameof(CodeFontFamily),
+            typeof(FontFamily),
+            typeof(MarkdownTextBlock),
+            new PropertyMetadata(new FontFamily("Consolas"), new PropertyChangedCallback(OnPropertyChangedStatic)));
```

Comment:
Agree and here is an example for setting default values outside of the code:
https://github.com/Microsoft/UWPCommunityToolkit/blob/dev/Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.xaml#L16

The idea is to use xaml instead of the PropertyMetadata field (which allows users to change it in a simple way)